developer/js/tests/test-parse-wordlist.ts

```diff
@@ -4,39 +4,39 @@ import 'mocha';
 import { makePathToFixture } from './helpers';
 
 const BOM = '\ufeff';
-const SENCOTEN_WORDLIST = [
-  ['TŦE', 13644],
-  ['E', 9134],
-  ['SEN', 4816],
-  ['Ȼ', 3479],
-  ['SW̱', 2621],
-  ['NIȽ', 2314],
-  ['U¸', 2298],
-  ['I¸', 1988],
-  ['ȻSE', 1925],
-  ['I', 1884]
-];
+const SENCOTEN_WORDLIST = {
```

Comment:
Is this now a `WordList` type?

Reply:
I guess a complication is we couldn't define a const that had identical wordforms (e.g.)
```
'I': 2, // from wordlist 1
'I', 5, // from wordlist 2
...
```

Reply:
Yep.  This matches the new format specified for `WordList` [here](https://github.com/keymanapp/keyman/pull/3340/files#diff-b0a389458ccb6df98f56754ae35edd35R7).

Reply:
> I guess a complication is we couldn't define a const that had identical wordforms (e.g.)

The final wordlist should never have two entries with the same wordform: that's the whole point of the parent PR 😄